screens/EventScreen/EventScreen.tsx

```diff
@@ -294,40 +297,40 @@ export default class EventScreen extends React.Component<Props, State>{
                       inputStyle={styles.groupNameInput} value={this.state.data.location}
                       isEditable={this.state.isEditable}></EditableLocation>
                   }
-                  <Text>Organizer</Text>
+                  <Text style={{ fontFamily: "Helvetica Neue, sans-serif", fontSize: 16, lineHeight: 23, color: "#333333", paddingBottom: 12 }}>Organizer</Text>
```

Comment:
Is this the right font?

Reply:
@GeorgeBellTMH This is one of the fonts listed on the mockups. Is it not supposed to be used? Let me know and I'll change it